.changeset/strange-starfishes-raise.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fuel-ts/account": patch
+"@fuel-ts/address": patch
+"@fuel-ts/program": patch
+"@fuel-ts/script": patch
+"fuels": patch
+---
+
+chore: added noImplicitOverride option to config
```

Comment:
@YaTut1901 It appears that we have 2 changeset files being added. We can safely remove `strange-starfishes-raise.md`. 

`large-points-arrive.md` seems to be already correct